test/worker_test.py

```diff
@@ -1483,26 +1482,6 @@ def complete(self):
 
 class MultipleWorkersTest(unittest.TestCase):
 
-    @unittest.skip('Always skip. There are many intermittent failures')
-    # This pass under python3 when run as `nosetests test/worker_test.py`
-    # but not as `nosetests test`. Probably some side effect on previous tests
-    @unittest.skipIf(six.PY3, 'This test fail on python3 when run with tox.')
```

Comment:
This is not strictly python3 related. It's flaky test that we can decide to keep or nor out side of this PR. I think it's fine we just remove this line and the comments above but still skip it.